settings.py

```diff
@@ -1,6 +1,5 @@
-TRACK_TERMS = ['#vegan', '#veganreceipe', '#veganfoodie', '#veganfood', '#vegancooking']
+TRACK_TERMS = []
+
+# Put your own keywords to filter the Twitter stream
+TRACK_TERMS = "KEYWORDS1","KEYWORDSn"
```

Comment:
let's move TRACK_TERMS in config.yaml (and remove `settings.py`) so we can have a central place where everything is configurable